spec/features/admin/admin_can_edit_user_profile_spec.rb

```diff
@@ -7,13 +7,13 @@
     @admin = create(:user, role: 2)
     login_as(@admin)
 
-    @user = create(:user, id: 100)
+    @user = create(:user, id: 100, email: 'user@email.com', slug: 'user-email-com')
```

Comment:
for future reference you should never inject ID values in test data... seed data is fine since it might come from CSV files or something